package/gluon-ebtables-net-rules/files/lib/gluon/ebtables/110-net-allow-ipv6

```diff
@@ -0,0 +1,8 @@
+siteConfig = require("gluon.site_config")
+
+rule ('NET_SRC -p IPv6 --ip6-src fe80::/10 -j RETURN')
```

Comment:
Now the additional multicast rule within our package makes sense. I have added it to this PR.